javascript/introduction-to-js-1/tasks/arrays/marking.md

```diff
@@ -70,4 +69,24 @@ myArray.forEach(function(element, index) {
 });
 
 let myString = myArray.join(' - ');
-```
\ No newline at end of file
+```
+
+## Task 4
+
+You should start by finding the index of ``"Eagles"``, using `birds.indexOf()`.
+
+Next, use the index with `birds.splice()` to remove the ``"Eagles"`` item.
+
+Finally, you can use `birds.filter()` to get an array containing only birds starting with "E".
+
+The function passed to `filter()` can use `bird.startsWith()` to test whether the bird starts with "E".
+
+```js
+const eaglesIndex = birds.indexOf("Eagles");
+birds.splice(eaglesIndex, 1);
+
+function startsWithE(bird) {
+  return bird.startsWith("E");
+}
+const eBirds = birds.filter(startsWithE).sort();
```

Comment:
Just to prove I'm paying attention: is the sort obligatory? Since the array is already sorted, I'm not sure if it adds much to include it.

Reply:
No, you're right, I don't know what I was thinking there. Looking at this PR I saw some other problems. There are two versions of each task, one that's supposed to run in the page and one that's supposed to be downloaded. These should stay in sync, and they weren't.

Also more `let`->`const`.